src/ensembl/src/content/app/custom-download/containers/content/CustomDownloadContent.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from './CustomDownloadContent.scss';
+import {
+  getSelectedTabButton,
+  getShowPreviewResult,
+  getPreviewResult
+} from '../../state/customDownloadSelectors';
+import { getAttributes } from './attributes-accordion/state/attributesAccordionSelector';
+
+import AttributesAccordion from './attributes-accordion/AttributesAccordion';
+import FiltersAccordion from './filter-accordion/FiltersAccordion';
+import TabButtons from './tab-buttons/TabButtons';
+import ResultHolder from './result-holder/ResultHolder';
+import { RootState } from 'src/store';
+import PreviewDownload from './preview-download/PreviewDownload';
+
+import { getSelectedAttributes, formatResults } from './result-holder/helpers';
+
+type Props = {
+  selectedTabButton: string;
+  showPreview: boolean;
+  previewResult: any;
```

Comment:
is it a verb (preview result)? is it a noun (preview of the result)? perhaps just `preview` is sufficient?